src/Microsoft.SqlTools.ServiceLayer/QueryExecution/Contracts/ExecuteRequests/ExecuteAndReturnResultRequest.cs

```diff
@@ -0,0 +1,54 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.QueryExecution.Contracts.ExecuteRequests
+{
+    /// <summary>
+    /// Parameters for executing a query from a document open in the workspace
+    /// </summary>
+    public class ExecuteAndReturnResultParams

```

Comment:
How about we call this `SimpleExecute` instead of `ExecuteAndReturnResult`? Something shorter at least?